wp-content/themes/credos/tpl_career.php

```diff
@@ -3,7 +3,7 @@
 Template Name: Career Page
 */
 ?>
-<?php get_header(); ?> 
+<?php get_header(); ?>  
```

Comment:
```suggestion
<?php get_header(); ?>
```
Remove trailing spaces everywhere